setup.cfg

```diff
@@ -8,20 +8,21 @@ cover-package = arrow
 cover-erase = true
 
 [coverage:run]
-branch = True
-source =
-    arrow
-    tests
```

Comment:
I would strongly suggest instrumenting coverage for your tests -- it helps you find bugs that accidentally turn off portions of your testsuite

Reply:
When I removed tests from the coverage run, the lines covered/hit did not change. Is it being instrumented incorrectly?